baybe/utils/lazy_loader.py

```diff
@@ -0,0 +1,14 @@
+""" A utility class for loading heavier modules lazily"""
+
+import importlib
+
+
+class LazyLoader:
```

Comment:
Please add docstrings to this class as well as its functions. Details on how to write the docstrings can be found in our contributing guidelines.